spec/models/resource_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Resource, type: :model do
+  let(:my_resource) { create(:resource) }
+
+  it { is_expected.to validate_presence_of(:title) }
+  it { is_expected.to validate_length_of(:title).is_at_least(5) }
+
+  describe "attributes" do
```

Comment:
you don't need tests like this since some other test will fail if a required field isn't present on the model